setup.py

```diff
@@ -18,13 +18,14 @@ def readme():
     'setuptools',
     'typeguard >= 2.1.1, < 3.0.0',
     'cryptography >= 1.4',
+    'gevent >= 1.1.2',
     # indirect dependency thorugh paramiko; just for version constraint (>=1.4)
     'paramiko >= 2.0.1',
     'Werkzeug >= 0.11',
     'oauthlib[rsa, signedtoken] >= 1.1.1, < 2.0.0',
     'Flask >= 0.10.1',
+    'Flask-Sockets >= 0.2.1',
     'apache-libcloud >= 1.1.0',
-    'waitress >= 1.0.2, < 2.0.0'
```

Comment:
Prerequisites in README of the *docker* branch does not mention individual dependencies but only Docker or a link to the separate document about barebone installation.